po/extract_pot_db

```diff
@@ -44,7 +44,7 @@ my $domain = $domain[0];
 # and ctx, which specifies a column to use for a msgctxt entry (unused at present).
 my @DBDEFS = (
               {'domain' => 'statistics', 'table' => 'statistics.statistic_event', 'columns' => ['title', 'description'], 'id' => 'date'},
-              {'domain' => 'countries', 'table' => 'area JOIN iso_3166_1 iso on iso.area = area.id', 'columns' => ['area.name'], 'comment' => ['iso.code']},
+              {'domain' => 'countries', 'table' => 'area JOIN iso_3166_1 iso ON iso.area = area.id', 'columns' => ['area.name'], 'comment' => ['iso.code']},
```

Comment:
I'm afraid this will cause issues with the .po(t) files, since they seem to include the db query? Hopefully it wouldn't cause new translations to be required, if so I'll drop this line.

Reply:
It just changes an optional comment that helps translators with locating the source string in the database. It won’t unmap translations.